packages/wdio-browserstack-service/src/testOps/usageStats.ts

```diff
@@ -0,0 +1,130 @@
+import FeatureStats from './featureStats.js'
+import FeatureUsage from './featureUsage.js'
+import { BStackLogger } from '../bstackLogger.js'
+import TestOpsConfig from './testOpsConfig.js'
+import type { TOUsageStats } from '../types.js'
+
+class UsageStats {
+    public static instance: UsageStats
+    public testStartedStats: FeatureStats
+    public testFinishedStats: FeatureStats
+    public hookStartedStats: FeatureStats
+    public hookFinishedStats: FeatureStats
+    public cbtSessionStats: FeatureStats
+    public logStats: FeatureStats
+    public launchBuildUsage: FeatureUsage
+    public stopBuildUsage: FeatureUsage
+
+    public static getInstance(): UsageStats {
+        if (!UsageStats.instance) {
+            UsageStats.instance = new UsageStats()
+        }
+        return UsageStats.instance
+    }
+
+    constructor() {
+        this.testStartedStats = new FeatureStats()
+        this.testFinishedStats = new FeatureStats()
+        this.hookStartedStats = new FeatureStats()
+        this.hookFinishedStats = new FeatureStats()
+        this.cbtSessionStats = new FeatureStats()
+        this.logStats = new FeatureStats()
+        this.launchBuildUsage = new FeatureUsage()
+        this.stopBuildUsage = new FeatureUsage()
+    }
+
+    public add(usageStats: UsageStats): void {
+        this.testStartedStats.add(usageStats.testStartedStats)
+        this.testFinishedStats.add(usageStats.testFinishedStats)
+        this.hookStartedStats.add(usageStats.hookStartedStats)
+        this.hookFinishedStats.add(usageStats.hookFinishedStats)
+        this.cbtSessionStats.add(usageStats.cbtSessionStats)
+        this.logStats.add(usageStats.logStats)
+    }
+
+    public getFormattedData(workersData: any[]) {
+        this.addDataFromWorkers(workersData)
+        const testOpsConfig = TestOpsConfig.getInstance()
+        const usage: TOUsageStats = {
+            enabled: testOpsConfig.enabled,
+            manuallySet: testOpsConfig.manuallySet,
+            buildHashedId: testOpsConfig.buildHashedId
+        }
+
+        if (!usage.enabled) {
+            return usage
+        }
+
+        try {
+            usage.events = this.getEventsData()
+        } catch (e) {
+            BStackLogger.debug('exception in getFormattedData: ' + e)
+
+        }
+        return usage
+    }
+
+    public addDataFromWorkers(workersData: any[]) {
+        try {
```

Comment:
There doesn't seem to be a reason for this `try/catch` block as there is nothing that could throw an error.

Reply:
Yes eemoved try catch from here